web/satellite/tests/unit/common/SearchComponent.spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import { mount, shallowMount } from '@vue/test-utils';
+import * as sinon from 'sinon';
+import SearchComponent from '@/components/common/SearchComponent.vue';
+
+describe('SearchComponent.vue', () => {
+    it('renders correctly', () => {
+        const wrapper = shallowMount(SearchComponent);
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('renders correctly with default props', () => {
+        const wrapper = shallowMount(SearchComponent);
+
+        expect(wrapper.find('input').props('placeHolder')).toMatch('');
+        expect(wrapper.find('input').props('search')).toMatch('');
+    });
+
+    it('functions onMouseEnter/onMouseLeave work correctly', () => {
+        const wrapper = mount(SearchComponent);
+
+        wrapper.vm.onMouseEnter();
+
+        expect(wrapper.vm.inputWidth).toMatch('602px');
```

Comment:
Seems like this test broke the build @VitaliiShpital . If we want to access private field, we need to use `vm.['inputWidth']`

Reply:
Do u mean that npm run build command fails?
its strange, because npm run test passes